conclusions/rq1.md

```diff
@@ -0,0 +1,6 @@
+## What can be learned from existing (non-software) communities?
+Looking at the different communities, it can be concluded that most of the communities have one thing in common: they work because they have a goal. In the case of the intentional communities: all but the practical communities have their ideals as a goal to fulfill. The practical communities lack a goal, but those communities work because they fulfill the goal of making life easier.  
+The goal of MMO-RGP communities are to make some part of the game-play possible or easier. It is because some quests can only be done in a party that forming one is a goal to achieve. Late in the game, when enemies can become exponentialy stronger, the necessity of trust-worthy party members becomes greater. This can be the reason and the goal to form a guild. A clear and good goal can make a guild stable and well-defined.  
```

Comment:
`the necessity of trust-worthy party members becomes greater`

I think necessity becomes higher and need becomes greater. Not sure though...